id: PYSEC-0000-CVE-2023-40267
details: 'GitPython before 3.1.32 does not block insecure non-multi options in clone
  and clone_from. NOTE: this issue exists because of an incomplete fix for CVE-2022-24439.'
affected:
- package:
    name: gitpython
    ecosystem: PyPI
    purl: pkg:pypi/gitpython
  ranges:
  - type: GIT
    repo: https://github.com/gitpython-developers/GitPython
    events:
    - introduced: "0"
    - fixed: ca965ecc81853bca7675261729143f54e5bf4cdd
  - type: ECOSYSTEM
    events:
    - introduced: "0"
    - fixed: 3.1.32
references:
- type: FIX
  url: https://github.com/gitpython-developers/GitPython/commit/ca965ecc81853bca7675261729143f54e5bf4cdd
- type: WEB
  url: https://github.com/gitpython-developers/GitPython/pull/1609
aliases:
- CVE-2023-40267
modified: "2023-08-11T12:58:00Z"
published: "2023-08-11T07:15:00Z"
